base/workflow/R/create_execute_test_xml.R

```diff
@@ -104,20 +105,19 @@ create_execute_test_xml <- function(model_id,
     pft <- dplyr::tbl(con, "pfts") %>%
       dplyr::filter(.data$modeltype_id == !!model.new$modeltype_id) %>%
       dplyr::collect()
+    
     pft <- pft$name[[1]]
     message("PFT is `NULL`. Defaulting to the following PFT: ",
             pft)
   }
-  if (length(pft) > 1) {
-    stop(
-      "Currently, only a single PFT is supported. ",
-      "Multiple PFTs will be implemented in a future version."
-    )
-  }
-  settings$pfts <- list(
-    pft = list(name = pft,
-               constants = list(num = 1))
-  )
+
+  ## Putting multiple PFTs separated by semicolon
+  settings$pfts <- strsplit(pft, ";")[[1]] %>%
+    purrr::map( ~ list(name = .x,
+                       constants = list(num = 1)
```

Comment:
Do we actually need this? I skimmed through `grep` output on our code and didn't see this as being essential anywhere, but I may be wrong. If the code works without it, I strongly suggest we drop it.

If it does turn out to be necessary, then at the _very_ least, if we do need it, we should make it increase sequentially by PFT; i.e.

```r
... %>% purrr::imap(~list(name = .x, constants = list(num = .y))
```

Reply:
`Do we actually need this?` 
By this you mean the support for multiple PFTs ?

Reply:
No, I mean `constants = list(num = 1)`.